src/VirtoCommerce.PricingModule.Web/Module.cs

```diff
@@ -88,16 +90,22 @@ public void Initialize(IServiceCollection serviceCollection)
                 configure.AddPolicy(typeof(ExportablePrice).FullName + "ExportDataPolicy", exportPolicy);
                 configure.AddPolicy(typeof(ExportablePricelistAssignment).FullName + "ExportDataPolicy", exportPolicy);
             });
+
+            serviceCollection.AddOptions<SimpleExportOptions>().Bind(Configuration.GetSection("Pricing:SimpleExport")).ValidateDataAnnotations();
```

Comment:
Please use this syntax instead of enforcing platform version bumping without necessary. 
serviceCollection.AddOptions<SimpleExportOptions>().Configure<IConfiguration>((opts, config) =>
            {
                opts.Bind(config.GetSection("Pricing:SimpleExport"));
            });